arch/arm/soc/arm/mps2/soc.h

```diff
@@ -7,6 +7,12 @@
 #ifndef _SOC_H_
 #define _SOC_H_
 
+#if defined(CONFIG_SOC_MPS2_AN521)
+#define __SAUREGION_PRESENT       1U        /* SAU regions present */
+#define __FPU_PRESENT             CONFIG_CPU_HAS_FPU
```

Comment:
Just wondering here: CPU_HAS_FPU is probably selected; does this mean it is is given the value "1" by default?

Reply:
Yes, it will be given the value of 1 or 0 being a bool Kconfig symbol